package.json

```diff
@@ -65,16 +63,17 @@
     "@nuxt/devtools-ui-kit": "^1.6.4",
     "@nuxt/eslint-config": "^0.7.3",
     "@nuxt/icon": "^1.10.2",
+    "@nuxt/kit": "^3.16.0",
     "@nuxt/module-builder": "^0.8.4",
-    "@nuxt/schema": "^3.14.1592",
+    "@nuxt/schema": "^3.16.0",
     "@nuxt/test-utils": "^3.15.1",
     "@release-it/conventional-changelog": "^9.0.3",
     "@tresjs/cientos": "^4.0.3",
     "@types/node": "^22.10.2",
     "@types/three": "^0.171.0",
     "changelogen": "^0.5.7",
     "eslint": "^9.17.0",
-    "nuxt": "^3.14.1592",
+    "nuxt": "^3.16.0",
     "playwright": "^1.49.1",
     "release-it": "^17.10.0",
     "three": "^0.171.0",
```

Comment:
nitpick: I saw that other packages are already using >=0.173 maybe it makes sense to bump this too :)